tests/TestCase/Utility/XmlTest.php

```diff
@@ -285,7 +285,7 @@ public function testFromArray()
         $obj = Xml::fromArray($xml);
         $this->assertTrue($obj instanceof \SimpleXMLElement);
         $this->assertEquals('tags', $obj->getName());
-        $this->assertCount(2, $obj);
+        $this->assertEquals(2, count($obj));
```

Comment:
Ideally, assertSame() to assert the same type, even though that's pretty obvious here.